src/main/webapp/script.js

```diff
@@ -100,21 +100,84 @@ window.addEventListener('load', () => {
 
   getOptions();
 
-  function initCalendar() {
-    new tui.Calendar('#calendar', {
-      defaultView: 'week',
-      useCreationPopup: true,
-      useDetailPopup: true,
-      disableDblClick: true,
-      disableClick: true,
-      isReadOnly: true,
-      scheduleView: ['time'],
-      taskView: false,
-      week: {
-        workweek: true,
-      },
+  if (document.head.id == 'scheduler') {
+    function initCalendar() {
+      new tui.Calendar('#calendar', {
+        defaultView: 'week',
+        useCreationPopup: true,
+        useDetailPopup: true,
+        disableDblClick: true,
+        disableClick: true,
+        isReadOnly: true,
+        scheduleView: ['time'],
+        taskView: false,
+        week: {
+          workweek: true,
+        },
+      });
+    }
+
+    initCalendar();
+  } else {
+    document.getElementById('submit-plan').addEventListener('submit', () => {
+      /**
+       * Gets results from /planner servlet to results
+       */
+      async function getPlan() {
+        const courseContainer = document.getElementById('order-area');
+        createSpinner(courseContainer);
+        const response = await fetch(
+            `/planner?selectedClasses=${selected.toString()}&semesters=${
+                document.getElementById('semesters').value}`);
+        courseList = await response.json();
+        createTable(courseList, courseContainer);
+      }
+
+      /**
+       * Creates spinner to signify loading
+       * @param {Object} courseContainer container for course list
+       */
+      function createSpinner(courseContainer) {
```

Comment:
maybe `attachNewSpinner`? The name `createSpinner` does not imply modification to any arguments.